README.md

```diff
@@ -42,6 +42,29 @@ module.exports = {
 };
 ```
 
+To correctly parse gjs/gts:
+Do not set an override for the parser in the global config.
+To use e.g. the typescript parser, the following should be used
+
+```js
+// .eslintrc.js
+module.exports = {
+  plugins: ['ember'],
+  // parser: '@typescript-eslint/parser', <-- needs to be removed, or set the gts/gjs one
+  // so we could also have:
+  // parser: "eslint-plugin-ember/gts-parser",
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/base', // or typescript-eslint/recommended, must come before plugin:ember/recommended
+    'plugin:ember/recommended', // or other configuration
+  ],
+  rules: {
+    // override / enable optional rules
+    'ember/no-replace-test-comments': 'error'
+  }
+};
```

Comment:
I'd prefer we specify: 
```
Overrides: [
{
  files: ["**/*.{gjs,gts}"],
  parser: 'ember',
}
]
```

Reply:
would this also be okay?
```js
{
      files: ['**/*.gts'],
      parser: 'eslint-plugin-ember/gts-parser',
    },
    {
      files: ['**/*.gjs'],
      parser: 'eslint-plugin-ember/gjs-parser',
    },
```

Reply:
Yeah

Reply:
@NullVoxPopuli 
with the non-breaking approach I also need to overwrite the preprocessor:
so better:
```js
overrides: [
    {
      files: ['**/*.gts'],
      extends: [
        'eslint:recommended-gjs',
      ]
    },
    {
      files: ['**/*.gjs'],
      extends: [
        'eslint:recommended-gjs',
      ]
    }
  ]
  ```

Reply:
This seems fine, but just wondering, is there a need or benefit to separating the parsers, or can they be specified as one? 

```
overrides: [
  {
    files: ["**/*.{gjs,gts}"],
    parser: 'eslint-plugin-ember/gjs-gts-parser',
  }
]
```

Reply:
we could.
we would need to detect if the extension is gts or gjs and choose by that, which internal parser should be chosen

Reply:
Would that be simple? Is there any reason why we shouldn't just auto-detect the extension for the user? If the user doesn't care about distinguishing between these file types, then we can avoid exposing that complexity to the user, and just handle it internally, right?